src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,52 @@
+.nav {
+  &__list {
+    display: flex;
+    line-height: 60px;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    margin: 0 10px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-align: center;
+    text-decoration: none;
+    color: $main-font-color;
+    height: 60px;
+    cursor: pointer;
+
+    &:hover {
+      color: $hover-color;
+    }
+  }
+}
+
+.is-active {
+  color: $hover-color;
+  position: relative;
```

Comment:
No need to set position only for `is-active` items - most probably it is better to have the same `position` for all items.